tests/robot-tests/tests/general_public/prod_table_tool.robot

```diff
@@ -0,0 +1,131 @@
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Further education" and text()="Further education"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Higher education" and text()="Higher education"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Pupils and schools" and text()="Pupils and schools"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section School and college outcomes and performance" and text()="School and college outcomes and performance"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section Teachers and school workforce" and text()="Teachers and school workforce"]
+
+    user checks page contains element
+    ...    //*[@data-testid="Expand Details Section UK education and training statistics" and text()="UK education and training statistics"]
+
+Check page contains correct publications
+    # Children's social care
+    user clicks element    //*[@data-testid="Expand Details Section Children's social care"]
+
+    user checks page contains element    //*[text()="Children in need and child protection"]
+    user checks page contains element    //*[text()="Children looked after"]
+
+    user checks page contains element    //*[text()="Children's social work workforce"]
+    user checks page contains element    //*[text()="Outcomes for children in social care"]
+
+    user checks page contains element    //*[text()="Secure children's homes"]
+    user checks page contains element    //*[text()="Serious incident notifications"]
+
+    # Covid 19
+    user clicks element    //*[@data-testid="Expand Details Section COVID-19"]
+
+    user checks page contains element    //*[text()="Attendance"]
+    user checks page contains element    //*[text()="Confirmed cases reported by Higher Education providers"]
+
+    user checks page contains element    //*[text()="Devices"]
+    user checks page contains element    //*[text()="Testing"]
+
+    # Destination of pupils & students
+    user clicks element    //*[@data-testid="Expand Details Section Destination of pupils and students"]
+
+    user checks page contains element    //*[text()="Destinations of key stage 4 and 16-18 pupils"]
+    user checks page contains element    //*[text()="NEET and participation"]
+
+    # Early years
+    user clicks element    //*[@data-testid="Expand Details Section Early years"]
+    user checks page contains element    //*[text()="Childcare and early years"]
+    user checks page contains element    //*[text()="Early years foundation stage profile"]
+
+    # Finance & funding
+    user clicks element    //*[@data-testid="Expand Details Section Finance and funding"]
+
+    user checks page contains element    //*[text()="Local authority and school finance"]
+
+    user checks page contains element    //*[text()="Student loan forecasts"]
```

Comment:
The spacing of these sections (i.e. if there is a blank line between keywords) is different for each theme - I'd make the spacing the same across every theme, just so its easier to read. 

I think my preference would be one space between clicking the element and checking for the first topic, and then no extra blank lines between keywords checking for other topics under that theme - as `Destination of pupils & students`. 

But it doesn't really matter how - making it consistent is the main thing.